nbconvert/preprocessors/regexremove.py

```diff
@@ -20,16 +20,14 @@ class RegexRemovePreprocessor(Preprocessor):
     of unicode strings. If the contents match any of the patterns, the cell
     is removed from the notebook.
 
-    By default, `patterns = [r'\Z']` which matches the empty string such that
-    strictly empty cells are removed. To modify the list of matched patterns,
+    To modify the list of matched patterns,
     modify the patterns traitlet. For example, execute the following command
-    to convert a notebook to html and remove cells containing only whitespace:
+    to convert a notebook to html and remove cells containing only whitespace::
```

Comment:
Small typo of an extra `:` here.

Reply:
This is actually reST syntax, see http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks

Without that, the formatting doesn't work, see the currently broken docs:
https://nbconvert.readthedocs.io/en/5.3.1/api/preprocessors.html#nbconvert.preprocessors.RegexRemovePreprocessor